addons/web/static/src/views/kanban/kanban_model.js

```diff
@@ -449,8 +461,6 @@ export class KanbanDynamicGroupList extends DynamicGroupList {
             this.model.notify();
```

Comment:
only catch NoTransactionInProgress here?

Reply:
for the part before commit (update,...)

Reply:
here both commit and abort must happen after start and we know that no other transaction for dataRecordId can have started and then be ended.